app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -74,7 +74,7 @@ class HomeActivityTest {
     launch(HomeActivity::class.java).use {
       // Wait until the expected text appears on the screen, and ensure it's for the welcome text view.
       waitForTheView(withText("Welcome back to Oppia!"))
-      onView(withId(R.id.welcome_text_view)).check(matches(withText("Welcome back to Oppia!")))
+//      onView(withId(R.id.welcome_text_view)).check(matches(withText("Welcome back to Oppia!")))
     }
   }
 
```

Comment:
Why do you need to make changes to `HomeActivityTest` ?

Reply:
initially i created dummy view call and implementation in this , now reverted the code